remote-payments-tester.php

```diff
@@ -25,6 +27,32 @@ public static function init() {
         self::initFreeExtensions();
     }
 
+    /**
+	 * Register the JS.
+	 */
+	public static function add_extension_register_script() {
+		if ( ! class_exists( 'Automattic\WooCommerce\Admin\Loader' ) || ! \Automattic\WooCommerce\Admin\Loader::is_admin_or_embed_page() ) {
```

Comment:
Can we remove this check?  Reason being is that we'll need access to this script registration via REST API and `is_admin_or_embed_page()` will be `false`.